.pre-commit-config.yaml

```
@@ -40,3 +40,11 @@ repos:
                               types-python-dateutil==2.9.0.20241206,
                               types-requests==2.32.0.20241016,
                               types-PyYAML==6.0.12.20240917]
+- repo: local
+  hooks:
+  - id: bqetl_format
+    name: bqetl_format
+    language: system
+    entry: script/bqetl
```

This will only work if the virtualenv is already activated.  Running the `./bqetl` script that auto-activates the virtualenv would work more consistently.
```suggestion
    entry: ./bqetl
```